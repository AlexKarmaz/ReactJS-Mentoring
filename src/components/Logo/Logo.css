.logo {
    color: F65261;
    font-size: xx-large;
}
