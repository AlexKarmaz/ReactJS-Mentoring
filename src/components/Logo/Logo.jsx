import React from 'react';
import './Logo.css';

const Logo = () => (
    <>
        <span className="logo"><b>netflix</b>roulette</span>
    </>
);

export default Logo
