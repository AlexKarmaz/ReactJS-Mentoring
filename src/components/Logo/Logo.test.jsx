import { shallow } from 'enzyme';
import React from 'react';
import Logo from './Logo';

describe('Logo', () => {
    it('should be rendered', () => {
        const logo = shallow(<Logo />);

        expect(logo.find('.logo').text()).toEqual('netflixroulette');
        expect(logo).toMatchSnapshot();
    });
});
