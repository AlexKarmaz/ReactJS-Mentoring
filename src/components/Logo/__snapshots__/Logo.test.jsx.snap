// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should be rendered 1`] = `ShallowWrapper {}`;
